CONTRIBUTING.md

```diff
@@ -8,10 +8,13 @@ Reading and following these guidelines will help us make the contribution proces
 
 * [Code of Conduct](#code-of-conduct)
 * [Getting Started](#getting-started)
+    * [Ideas](#ideas)
     * [Discussions](#discussions)
     * [Issues](#issues)
     * [Pull Requests](#pull-requests)
       * [Git Rebase Workflow](#git-rebase-workflow)
+    * [Show and Tell](#show-and-tell)
+    * [Development Process](#development-process)
```

Comment:
Many of the table of contents items don't match the section names, and therefore these links don't work

Reply:
fixed